crypto/build.gradle.kts

```
@@ -15,6 +15,9 @@ dependencies {
   implementation("org.bouncycastle:bcprov-jdk15on:1.70")
   implementation("org.bouncycastle:bcpkix-jdk15on:1.70")
   implementation(project(":common"))
+
+  implementation("com.amazonaws:aws-java-sdk-kms:1.12.538")
```

It seems like AWS specific definitions (for example, `AWSKMS` as the constructor parameter) are needed for client to be able to call this. Thus, it's making part of the API contract, and you'd need to change this. 
```suggestion
  api("com.amazonaws:aws-java-sdk-kms:1.12.538")
```